openapi.yaml

```diff
@@ -7820,7 +7820,9 @@ paths:
       - Networking
       summary: List IPv6 Pools
       description: >
-        Displays the IPv6 pools on your Account.
+        Displays the IPv6 pools on your Account. A pool of IPv6 addresses are routed to all of your
+        Linodes in a single Region. Any Linode on your Account may bring up any address in this
```

Comment:
NB: I could see a benefit of linking the word Region to the LIST Regions endpoint.

Reply:
Added!